dev-docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3"
```

Comment:
can we use more specific version? `3.8` seems to be the newest one

Reply:
Done